lib/rules/best-practises/no-comparison-in-expression.js

```diff
@@ -0,0 +1,82 @@
+const BaseChecker = require('../base-checker')
+const { severityDescription } = require('../../doc/utils')
+
+const ruleId = 'no-comparison-in-expression'
```

Comment:
the rule name feels wrong, since it's perfectly valid to have a comparison in an _expression_. All comparisons are expressions, actually :sweat_smile: the problem is when it's the only expression in an expression statement, which is a workaround of all non-pretentious programming languages for converting an expression (something meant to return a value (and, platonically, _only_ returning a value)) into a statement (something executed for its side effects). A better rule name would be `no-comparison-statement`, but open to suggestions regardless